content/docs/deployment/clients/_index.md

```diff
@@ -188,6 +188,47 @@ The service can be uninstalled with the following command:
 
 Confirm with `ps -eaf | grep velo` and `sudo launchctl list | grep velo`.
 
+### Linux
+
+The dpkg (Debian)or rpm (Red Hat/CentOS/Fedora) tools can be used to install Velociraptor on
+Linux clients after creating an appropriate package. The following command installs binary & config to /usr/local/sbin.
+Persistence is via launchd (check with `ps -eaf | grep velo` and
+`sudo launchctl list | grep velo`)
+
+#### Debian Package
+Create
+```shell
+# velociraptor --config client.config.yaml debian client
+```
+Install
+```shell
+# sudo dpkg -i velociraptor.deb
+```
+Remove
+```shell
+# sudo apt-get remove velociraptor
```

Comment:
The client package is called velociraptor_client

Reply:
Updated should be clearer now.